File: 01-business-sub-system/src/main/java/com/github/yingzhuo/datawarehouse/businesssubsys/dao/OrderItemDao.scala
package com.github.yingzhuo.datawarehouse.businesssubsys.dao

import com.github.yingzhuo.datawarehouse.businesssubsys.domain.OrderItem
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
trait OrderItemDao extends JpaRepository[OrderItem, String] {
}

File: 01-business-sub-system/src/main/java/com/github/yingzhuo/datawarehouse/businesssubsys/dao/OrderItemDao.java
package com.github.yingzhuo.datawarehouse.businesssubsys.dao;

import com.github.yingzhuo.datawarehouse.businesssubsys.domain.OrderItem;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface OrderItemDao extends JpaRepository<OrderItem, String> {
}
